trunk/src/app/srs_app_gb28181.cpp

```diff
@@ -421,12 +425,12 @@ srs_error_t SrsGbListener::initialize(SrsConfDirective* conf)
 
     bool sip_enabled = _srs_config->get_stream_caster_sip_enable(conf);
     if (!sip_enabled) {
-        return srs_error_new(ERROR_GB_CONFIG, "GB SIP is required");
+        srs_warn("GB SIP is disabled.");
```

Comment:
one more thing to review. About the conf directives of `sip.enable`:
https://github.com/ossrs/srs/blob/ea7e2c28491ae61cc1b7aaa207ce38a74fa36487/trunk/conf/full.conf#L681-L683

The code here means: `GB internal SIP is disabled`, so don't break the GB process, SRS can still support the external SIP. 

So, there are a few thing to consider:
1. improve the log: `GB [internal|embedded] SIP is disabled`;
2. Internal SIP is no longer a necessary one, so use `srs_trace` to replace `srs_warn`;
3. Maybe rename `sip_enable` to `internal_sip_enabled`;
4. External sip is the future, so no need to enable or disable it?

Reply:
In essence, the external SIP server is not part of SRS. You can look at my implementation, even use it directly, but it's not mandatory. So, I don't want to modify it for now.